### DISCUSSION TITLE
How to "login" from command line?

### DISCUSSION THREAD

**Discussion by Coder3333** [ariga/atlas#2856] [Q&A]
We have to perform an "atlas login" before we can use advanced schema inspection features; however, this command takes the user to a browser to perform the login action.  Is there a way to provide all criteria to the login command on the command line, so the browser does not need to be invoked?  Thanks.

**Comment by a8m** [comment]
In order to login in CI/CD environments, you can login with a `--token` flag. See this guide how to set it up: https://atlasgo.io/cloud/bots
